Add Job Guard to each integration job script (#1091)
kyma-project/test-infra: prow/scripts/cluster-integration/README.md

- Test the Kyma installation.
- Clean up all provisioned resources, also if you get an error.

> > **NOTE:** Run [Job Guard](./../../../development/tools/cmd/jobguard/README.md) at the beginning of each integration job that builds and installs Kyma witch changes from the pull request.

Reviewer: ```suggestion
> **NOTE:** Run [Job Guard](./../../../development/tools/cmd/jobguard/README.md) at the beginning of each integration job that builds and installs Kyma with changes from the pull request.
```

> > **NOTE:** [Job Guard](./../../../development/tools/cmd/jobguard/README.md) must be execute at the beginning of each integration job that builds and installs Kyma witch changes from the pull-request.

Reviewer: ```suggestion
> **NOTE:** Run [Job Guard](./../../../development/tools/cmd/jobguard/README.md) at the beginning of each integration job that builds and installs Kyma witch changes from the pull request.
```
